playwright/tests/integration/properties/layout.spec.ts

```diff
@@ -87,7 +87,7 @@ test('extract Dialog', async ({ page }) => {
   await expect(page.getByRole('dialog')).toBeVisible();
   const name = page.locator('.extract-dialog-name input');
   const namespace = page.locator('.extract-dialog-namespace input');
-  const dataclass = page.locator('.extract-dialog-dataclass button');
+  const dataclass = page.locator('.extract-dialog-dataclass button').first();
```

Comment:
not new, but why not be more clear here?
1. the locators should be scoped to the dialog
2. the input fields have a label, why not use it for the selector, then you can remove this class: `dialog.getByRole('textbox', {name: 'Name'})`
3. also the button has a name: `dialog.getByRole('button', {name: '...'})`

this makes reading and debuing such test much easier...